src/main/java/uk/gov/justice/hmpps/prison/repository/jpa/repository/AdjudicationRepository.java

```diff
@@ -15,4 +17,7 @@
 
     Optional<Adjudication> findByParties_AdjudicationNumber(final Long adjudicationNumber);
     List<Adjudication> findByParties_AdjudicationNumberIn(final List<Long> adjudicationNumbers);
+
+    @Query("SELECT DISTINCT a FROM Adjudication a JOIN a.parties p WHERE (:agencyLocationId IS null Or a.agencyLocation.id = :agencyLocationId) AND (p.adjudicationNumber IN :adjudicationNumbers)")
```

Comment:
This is done so we can allow passing in of a null agencyId and get results back for all agencies - which doesn't work with using standard spring data naming conventions.